### ISSUE TITLE
pytorch implementation of masked CRF

### ISSUE THREAD

**Issue by ZenBel** [zhw666888/Pytorch-MCRF#2]
Hello, and thank you for your `pytorch` implementation of masked CRF. I have a question concerning this part of the code:

```
for i in range(self.mask_tran_matrix.shape[0]):
    for j in range(self.mask_tran_matrix.shape[1]):
        if self.mask_tran_matrix[i][j] == -10.0:
            self.transitions.data[i][j] = torch.tensor([-1.0])
```

Why do you assign `torch.tensor([-1.0])` instead of `torch.tensor([-10.0])`?

**Comment by zhw666888** [comment]
Thank you for your question,this part is wrong,the new version has been updated,thanks for your attention!
